pom.xml

```diff
@@ -232,6 +232,10 @@
                             <name>jboss.remoting.debug-buffer-leaks</name>
                             <value>${leak.debug}</value>
                         </property>
+                        <systemPropertyVariables>
+                         <user.language>en</user.language>
```

Comment:
I am not commenting on the change itself, I just stumbled upon this looking for something else :-)